manifest_staging/charts/secrets-store-csi-driver/templates/secrets-store-csi-driver-windows.yaml

```diff
@@ -176,6 +178,10 @@ spec:
           hostPath:
             path: {{ .Values.windows.providersDir }}
             type: DirectoryOrCreate
+        {{ range .Values.windows.additionalProvidersDirs }}
```

Comment:
Could we do `{{ range $i, $path :=  .Values.windows.additionalProvidersDirs }}` and append the index to the name of the host path?

Reply:
i think i finally got the syntax correct for this, thanks!